ddtrace/settings/_config.py

```diff
@@ -849,9 +849,11 @@ def _convert_rc_trace_sampling_rules(
         """
         rc_rules = self._remove_invalid_rules(rc_rules)
         for rule in rc_rules:
-            tags = rule.get("tags")
-            if tags:
-                rule["tags"] = self._tags_to_dict(tags)
+            if "tags" in rule:
```

Comment:
This fix will be released/documented in this PR: https://github.com/DataDog/dd-trace-py/pull/13245